tests/framework/ROM/TimeSeries/ARMA/multiple_fourier.xml

```diff
@@ -72,16 +71,21 @@
       <Features>scaling</Features>
       <seed>84</seed>
       <pivotParameter>Time</pivotParameter>
-      <Pmax>3</Pmax>
-      <Pmin>3</Pmin>
-      <Qmax>1</Qmax>
-      <Qmin>1</Qmin>
       <SpecificFourier variables="A,B">
         <periods>20,100</periods>
         <orders>1,1</orders>
       </SpecificFourier>
-      <Fourier>10,50</Fourier> <!-- will apply to the rest (C,D) -->
+      <Fourier>10,50</Fourier>
+      <!-- will apply to the rest (C,D) -->
       <FourierOrder>1,1</FourierOrder>
+      <P>3</P>
+      <Q>1</Q>
+      <P>3</P>
+      <Q>1</Q>
+      <P>3</P>
+      <Q>1</Q>
+      <P>3</P>
+      <Q>1</Q>
```

Comment:
1. why P and Q and duplicated here
2. TestInfo Should be updated.

Reply:
fixed